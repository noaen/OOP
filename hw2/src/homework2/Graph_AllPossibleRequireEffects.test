# Sample Testing file

# Create a graph
CreateGraph graph1

# Add null node
AddNode graph1 

# Create nodes
CreateNode n1 5
CreateNode n2 5
# Add nodes
AddNode graph1 n1
AddNode graph1 n2
# Add existing node
AddNode graph1 n1

# Add an edge
AddEdge graph1 n1 n2
# Add an existing edge
AddEdge graph1 n1 n2
# Add an edge to a node not in graph
CreateNode n3 5
AddEdge graph1 n1 n3
