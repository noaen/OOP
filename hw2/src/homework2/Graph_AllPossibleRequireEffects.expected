# Sample Testing file

# Create a graph
created graph graph1

# Add null node
Exception: homework2.CommandException: Bad arguments to addNode: [graph1]

# Create nodes
created node n1 with cost 5
created node n2 with cost 5
# Add nodes
added node n1 to graph1
added node n2 to graph1
# Add existing node
node n1 already exists in graph graph1

# Add an edge
added edge from n1 to n2 in graph1
# Add an existing edge
edge from n1 to n2 already exists in graph graph1
# Add an edge to a node not in graph
created node n3 with cost 5
node n3 does not exist in graph graph1
